spec/support/serializers/firm_serializer.rb

```diff
@@ -0,0 +1,103 @@
+class FirmSerializer < ActiveModel::Serializer
```

Comment:
Placeholder for the spike: We discussed about using stubs for the ES API removing the need for these extra objects